website/blog/2023-02-03-product-update.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "February 2023 dbt Update"
+description: "The latest product updates from dbt Labs"
+slug: dbt-product-update-2023-february
+# authors: [lauren_craigie] 
+
+tags: [dbt updates]
+hide_table_of_contents: false
+
+date: 2023-02-03
+is_featured: false
+---
+Last month I asked my group chat what date was *too* late to open work conversations with "Happy New Year." Most said January 15th. No one said anything close to February 2nd/3rd but here we are... Happy New Year! 🎉 It’s been awhile since I’ve last seen you, and there’s lots to share. Read on for the latest in dbt Cloud + new tools and resources.
+
+<!--truncate-->
+
+## **What's new**
+
+*What you might have missed this quarter*
+
+- **[dbt utils v1.0 is live](https://docs.getdbt.com/guides/migration/versions/upgrading-to-dbt-utils-v1.0).** Pay close attention to the changes in surrogate key generation, and don’t miss [this developer blog post](https://docs.getdbt.com/blog/grouping-data-tests) from dbt utils contributor Emily Riederer discussing the new ability to run grouped checks.
+- **[dbt-databricks adapter is now available in dbt Cloud](https://docs.getdbt.com/guides/migration/tools/migrating-from-spark-to-databricks#migration)**. The new adapter that dropped in November is easier to set up, has better defaults for developing on Databricks, and supports Unity Catalog (on dbt v1.1+).
+- **[dbt natively supports Python.](https://www.getdbt.com/blog/introducing-support-for-python/)** As of October, you can deploy Python models alongside your SQL ones, enjoying a single view of lineage across both.
+- **[dbt Semantic Layer hit Public Preview](https://docs.getdbt.com/docs/use-dbt-semantic-layer/dbt-semantic-layer)** for Snowflake users in October. Centrally define metrics in your dbt project, and enable anyone to query them from their preferred Semantic Layer integrated tooling.
+
+![https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png](https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%209.55.51%20PM.png)
```

Comment:
This is pic renders in low quality; can you potentially upload it as an actual image into the docs repo and use a lightbox element (that way you can add alt text description to this)? See [Doug's autoscaling CI blog post for an example](https://raw.githubusercontent.com/dbt-labs/docs.getdbt.com/current/website/blog/2023-01-27-autoscaling-ci.md).